test/elm/external/external-test.ts

```diff
@@ -15,7 +15,11 @@ describe('Retrieve', () => {
     const findRecordsSpy = sinon.spy(this.ctx, 'findRecords');
     this.conditions.exec(this.ctx);
 
-    findRecordsSpy.should.be.calledWithExactly(this.conditions.datatype, this.conditions);
+    const expectedRetrieveDetail: RetrieveDetails = {
+      datatype: '{https://github.com/cqframework/cql-execution/simple}Condition'
+    };
```

Comment:
This is the simplest of retrieves.  If it's not too much trouble, it would be nice to have a test that verifies that the `RetrieveDetails.code` property is correctly set to a calculated value (either a `Code[]` or `ValueSet`). Extra bonus points for a test that does the same for `RetrieveDetails.dateRange`